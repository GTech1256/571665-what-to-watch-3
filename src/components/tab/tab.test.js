import React from "react";
import renderer from "react-test-renderer";
import Tab from "./tab";

const mock = {
  title: `title`,
  isActive: true,
  onClick: () => {}
};

it(`<Tab /> shoult render correct`, () => {
  const tree = renderer.create(
      <Tab {...mock} />
  );

  expect(tree).toMatchSnapshot();
});
