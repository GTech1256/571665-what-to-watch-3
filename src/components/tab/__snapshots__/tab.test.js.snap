// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab /> shoult render correct 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
  >
    title
  </a>
</li>
`;
